README.md

```diff
@@ -3,10 +3,11 @@
 # Installing preliminary Conda packages
 
 Some preliminary Conda packages can be installed as so. Replace `<CUDA
-version>` with either `9.2` or `10.0`.
+version>` with either `9.2`, `10.0`, or `10.1`. Also replace `<processor type>`
+with `cpu` or `gpu`.
 
 ```
-conda create -n ucx -c conda-forge -c jakirkham/label/ucx cudatoolkit=<CUDA version> ucx-proc=*=gpu ucx ucx-py python=3.7
+conda create -n ucx -c conda-forge -c jakirkham/label/ucx-new cudatoolkit=<CUDA version> ucx-proc=*=<processor type> ucx ucx-py python=3.7
```

Comment:
Does `cudatoolkit` package now depend on the ghost `cuda` package added somewhere around conda 4.7?

Reply:
AFAIK no. Please see issue ( https://github.com/conda/conda/issues/9115 ) for some discussion on this topic.